builtin/types/chainconfig/chainconfig.proto

```diff
@@ -97,11 +98,11 @@ message Action {
 }
 
 message AppStoreConfig {
-    uint64 num_evm_keys_to_prune = 1;
+    uint64 num_evm_keys_to_prune = 1 [(gogoproto.jsontag) = "num_evm_keys_to_prune"];
 }
 
 message EvmConfig {
-    uint64 gas_limit = 1;
+    uint64 gas_limit = 1 [(gogoproto.jsontag) = "gas_limit"];
```

Comment:
What's this for?

Reply:
To remove `omitempty` tag as we are parsing this protobuf to genesis json. With `omitempty` tag, 0 value will not be written out.